src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+
+  display: flex;
+  flex-direction: column;
```

Comment:
This styles are redundant, all elements in your card are blocks and they started from the new line by default